### ISSUE TITLE
Next round Epic

### ISSUE THREAD

**Issue by albinr99salt** [albinr99salt/no-name#72]
## Progress to next round

After a winner has been selected, the game should progress to the next round. All players should receive the correct amount of new cards. A new cardwizz should be also be appointed.

## User Story
As a user, when the winner of a round has been selected and announced, i want the game to progress to the next round. I want to clearly understand that a new round has begun.
I want to be able to see who won before the game switches to next round.
I want to be given new card(s) so that i have the correct amount of cards in my hand.
I want it to be a new cardwizz for the new round and i want a new black card to appear. 

## Requirements
- [x] switch to new round when round is complete
- [x] wait before switch so that everyone get a chance to see the winner, and the winning card
- [x] hand out new cards to all players when a new round begin
- [x] all players should have the correct amount of cards after the new round has begun
- [x] there should be a new cardwizz in the new round
- [x] there should be a new black card in the new round
- [x] it should be clear that a new round has started

### Backend
- [x] wait a set time before going to next round
- [x] set played cards to state `used`
- [x] update current round on game
- [x] update card wizz on game
- [x] update black card on game
- [x] hand out new cards to all players
- [x] reset players has_played flag

**Comment by albinr99salt** [comment]
After some thinking i think issue #80 have to be completed before this can be done.

**Comment by albinr99salt** [comment]
Seems like there is possibly some typeorm issue with relations with composite keys.

Consider setting up the relation between game and currentRound differently.

GameRound have the following columns
id, game_key, round_number.
we could make the relation between a game and the "currentRound" go by the id of the gameRound table instead. Skipping the composite key step, and just using a foreign key in the game table.

This way we could just update the relation and we should be good to go.

**Comment by albinr99salt** [comment]
Above solution seem to have solved the issue. Consider reproducing in a very simpel model and report to typeorm as a bug.
